english_to_morse_code.rb

```diff
@@ -0,0 +1,20 @@
+def decode_char(alphabet)
+  morse = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F',
+    '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L',
+    '--' => 'M', '-.' => 'N', '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R',
+    '...' => 'S', '-' => 'T', '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X',
+    '-.--' => 'Y', '--..' => 'Z'
+  }
+  morse[alphabet]
+end
+
+def decode_word(str)
```

Comment:
- [ ] The showing result its still not as expected: 
![Screenshot from 2021-12-16 20-19-59](https://user-images.githubusercontent.com/62705947/146435079-47766ade-1fc3-459d-b440-29674f6e198b.png)

Please refer to the expected result from the previous reviewer